<launch>
    <arg name="markerId"        default="0"/>
    <arg name="markerSize"      default="0.1575"/>    <!-- in m -->
    <arg name="camera_info"    default="/camera/color/camera_info"/>
    <arg name="image_topic"    default="/camera/color/image_raw"/>
    <arg name="marker_frame"    default="aruco_marker_frame"/>
    <arg name="camera_frame"    default="aruco_camera_optical_frame"/>
    <arg name="ref_frame"       default=""/>  <!-- leave empty and the pose will be published wrt camera_frame -->

    <node pkg="localizer" type="localizer" name="single_localizer">
        <remap from="/camera_info"      to="$(arg camera_info)" />
        <remap from="/image"            to="$(arg image_topic)" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="marker_id"          value="$(arg markerId)"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>   <!-- frame in which the marker pose will be refered -->
        <param name="camera_frame"       value="$(arg camera_frame)"/>
        <param name="marker_frame"       value="$(arg marker_frame)" />
    </node>

</launch>